packages/pages/src/util/runtime.test.ts

```diff
@@ -168,3 +96,45 @@ describe("runtime", () => {
     global.process = originalProcess;
   });
 });
+
+const setupMockEnvironment = (
+  simulatedEnvironment?: "node" | "deno"
+): NodeJS.Process => {
+  const originalProcess = process;
+  (global.process as any) = undefined;
+
+  if (simulatedEnvironment === "deno") {
+    global.Deno = {
+      version: {
+        deno: "1.24.0",
+      },
+    };
+    (browserOrNode.isDeno as any) = true;
+    (browserOrNode.isNode as any) = false;
+  }
+
+  if (simulatedEnvironment === "node") {
+    global.process = {
+      ...originalProcess,
+      versions: {
```

Comment:
this might be a dumb question but why do we need to add these to global.process?

Reply:
Because the class will look for it on line 14